week 2/script.js

```diff
@@ -15,7 +15,58 @@ class CurrencyRate {
     }
 }
 
+
+async function getData() {
+  try {
+    const response = await fetch(
+      "https://raw.githubusercontent.com/SathiC1990/sathic1990/main/data/currency.json"
+    );
+
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`);
+    }
+
+    const myData = await response.json();
+    const currencyRates = myData;
```

Comment:
You have already defined a `currencyRates` variable at the beginning of this file. It is a global variable. This one (on line 30) is scoped in the `try` statement. To avoid confusion, I would suggest renaming one of these `currencyRates` variables or using only one.